### DISCUSSION TITLE
[config help] Renovatebot ignore dynamic configuration

### DISCUSSION THREAD

**Discussion by ravecat** [renovatebot/renovate#17454] [Request Help] [answered]
Try to use renovate as [child triggered pipeline](https://gitlab.com/rvct/templates/-/blob/1761f3c71c1a82eeb53686a2d2872306a60072cc/vulnerability/renovatebot/renovatebot.gitlab-ci.yml) for gitlab and set [default config](https://gitlab.com/rvct/templates/-/blob/1761f3c71c1a82eeb53686a2d2872306a60072cc/vulnerability/renovatebot/renovatebot.gitlab-ci.yml#L30). It will be useful, when renovatebot enabled for many repos and we want to use similar config everwhere without copy same structure

the logic is simple, if a bot is used in the repository, but there is no config, use the default config 

```
export RENOVATE_CONFIG_FILE="$PWD/$TEMPLATES_ARTIFACTS_DIR/vulnerability/renovatebot/renovate.json"
LOG_LEVEL=debug renovate --print-config true $RENOVATE_EXTRA_FLAGS
```

but renovatebot checks standart [configuration list ](https://docs.renovatebot.com/configuration-options/#configuration-options) and [skips repo without standart config source](https://gitlab.com/rvct/templates/-/jobs/2945029487#L380)

Any workaround to resolve it?

**Comment by Gabriel-Ladzaretti** [comment, marked as answer]
As i understand it, `RENOVATE_CONFIG_FILE` sets the `Global` [config](https://docs.renovatebot.com/getting-started/running/#global-config) needed for self hosted users, repos are being ignore because they missing the repository config. try setting the `requireconfig` option in your global config and possibly `onboarding` as well.
https://docs.renovatebot.com/self-hosted-configuration/#requireconfig
https://docs.renovatebot.com/self-hosted-configuration/#onboarding

**Reply by ravecat** [reply]
@Gabriel-Ladzaretti yep! with [combination](https://gitlab.com/rvct/templates/-/blob/5733ef443e8c4b6829c79dabd9f0ee2387dba076/vulnerability/renovatebot/renovatebot.gitlab-ci.yml) `RENOVATE_CONFIG_FILE` and `RENOVATE_REQUIRE_CONFIG="ignored"` we can achieved required behavior
